docs/modules/jenkins-shared-library/pages/odsComponentStageUploadToNexus

```diff
@@ -0,0 +1,19 @@
+:page-partial:
+
+Triggers the upload of an artifact to Nexus
+
+Available options:
+
+[cols="1,2"]
+|===
+| Option | Description
+
+| distributionFile
+| default is `${projectId}-${componentId}.tar.gz`
+
+| repoType
+| default is `candidates`
+
+| groupId
+| default is the `groupId` on project level, or in case not set at all `org.opendevstack.${projectId}`
+|===

```

Comment:
Could you move this to the `partials` folder? (following a recent update)
Also, there needs to be reference from the `component-pipeline.adoc` in order for this partial to show up.